build.sh

```diff
@@ -29,6 +29,8 @@ PRINT_USAGE() {
     echo "  -j [N], --jobs[=N]   Multicore build, allow N jobs at once"
     echo "  -n, --ninja          Build with ninja instead of make"
     echo "      --xcode          Generate XCode project"
+    echo "      --gen[=Gen]      Give a custom generator to cmake"
+    echo "      --make[=cmd]     Give a custom make command"
```

Comment:
'--gen' part I understood but not sure if we ever need '--make' argument. A bit confusing though